libs/generators/src/lib/generators/generators-menu/generators-menu.component.spec.ts

```diff
@@ -1,14 +1,22 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { Router } from '@angular/router';
 
 import { GeneratorsMenuComponent } from './generators-menu.component';
 
 describe('GeneratorsMenuComponent', () => {
   let component: GeneratorsMenuComponent;
   let fixture: ComponentFixture<GeneratorsMenuComponent>;
+  let routerMock: any;
 
   beforeEach(async () => {
+    routerMock = {
+      navigate: () => null,
+    };
+
     await TestBed.configureTestingModule({
       imports: [GeneratorsMenuComponent],
```

Comment:
https://angular.io/api/router/testing/RouterTestingModule

```suggestion
      imports: [GeneratorsMenuComponent, RouterTestingModule],
```